# Introduction

These pages will contain documentation for the Helium pedigree visualization software developed by Information and Computational Sciences at the James Hutton Institute in Scotland.

Please either click on the 'Wiki' link above or [here](https://github.com/cardinalb/helium-docs/wiki) to go to the documentation. These documents are very much a work in progress so if there are any questions you may have on Helium then please contact us on [helium@hutton.ac.uk](mailto:helium@hutton.ac.uk). Also, any feedback on how we can improve Helium or add new features that would be useful to the community please let us know.

# About Helium
The ability to predict and visualize the inheritance of genes that facilitate resistance to pathogens or any other commercially important characteristic is crucially important to experimental plant genetics and commercial plant breeding programmes. Derivation of this information by traditional molecular techniques is expensive and time consuming, even with developments in high-throughput technologies over the past five years. This is especially true in industrial settings where, due to time constraints relating to growing seasons, many thousands of plant lines may need to be screened quickly, efficiently and economically every year.

![Helium Screenshot](/images/helium2.png)*Helium allows, in this case barley DUS characters, to be colour coded and viewed in a pedigree context.*

There is, however, a cognitive limitation conceptualising large pedigree structures. While it may not be achievable or indeed necessary to understand every mating relationship between related individuals or lines, an overall picture can lead to insight into the data and any patterns it may contain. This can also aid in the identification of problems within datasets when coupled with expert domain knowledge. Tools that provide the ability to pull these data types together, and integrate with pedigree structure, will facilitate breeders and researchers to make more informed decisions relating to suitable plant lines to use in subsequent crossings. It is imperative that we develop more suitable crops to meet both the population imposed mandate for increased yield/production and adaptation to a changing global climate.

![Helium Screenshot](/images/helium3.png)*Helium using genetic similarity data calculated from 7,000 SNP markers. The darker the colour the greater the similarity to a selected reference variety. The histogram shows the distribution of values round the reference line.*

Our aim is to make Helium a generic platform in which various data types can be shown in a pedigree context and we hope to have a beta version available soon that users can download and experiment with. We have also modified the Germinate database platform to handle pedigree data in a new format which makes it suitable for inclusion into Helium.

**Helium won best paper award at BioVis 2014 in Boston, USA.**

Shaw, P.D., Kennedy, J., Graham, M., Milne, I. and Marshall, D.F. 2014. Helium: Visualization of Large Scale Plant Pedigrees. BMC Bioinformatics. 15:259. [DOI: 10.1186/1471-2105-15-259](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-259)

Helium started as part of my PhD along with Jessie Kennedy and Martin Graham from the Institute for Informatics and Digital Innovation at Edinburgh Napier University but is now maintained by the [James Hutton Institute Information and Computational Sciences Group](https://www.hutton.ac.uk/research/groups/information-and-computational-sciences).

If you are interested in using or working with us on the Helium platform then please contact us at [helium@hutton.ac.uk](mailto:helium@hutton.ac.uk).
